### DISCUSSION TITLE
the-finale/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#10] [Announcements]
# the-finale/

Last Post Of The Year

https://www.supergoodcode.com/the-finale/

**Comment by bjb43** [comment]
Hi, thanks for this post and also providing the precompiled library. How can I check if the game is actually using nine and not simply sticking with vulkan when launched like D_PRELOAD=/path/to/nine_sdl.so %command% -vulkan? Is there any console output etc I could look out for?

**Comment by Jaffies** [comment]
Hi, recently valve posted an update to CSS/HL2/other stuff that enables dxvk stuff and made vulkan renderer by default.
Unfortunately, they work on steam runtime 3.0 and not the legacy one. Is there a way to make it work?
